src/OpenTelemetry.Instrumentation.AWSLambda/AWSLambdaInstrumentationOptions.cs

```diff
@@ -10,6 +10,7 @@ public class AWSLambdaInstrumentationOptions
 {
     /// <summary>
     /// Gets or sets a value indicating whether AWS X-Ray context extraction should be disabled.
+    /// If set to true, the AWS X-Ray trace header will be ignored. Otherwise, the propagation context will be extracted from the trace header automatically generated by AWS X-Ray.
```

Comment:
Does this add any new information beyond what the first sentence already states?

The only thing is the "automatically generated by AWS X-Ray" bit but that is a bit confusing without further elaboration (which is provided with the Note you added in the README)

Reply:
The main objective is adding more information about the "context extraction" and how it happens.

But - as you stated, and I agree with it - it is confusing without the explanation provided in the README file.
Probably because the explanation is out of place.

So, removing this sentence, ok?

Reply:
Done!